x/subspaces/keeper/groups.go

```diff
@@ -106,6 +111,9 @@ func (k Keeper) DeleteUserGroup(ctx sdk.Context, subspaceID uint64, groupID uint
 
 // AddUserToGroup adds the given user to the group having the provided id inside the specified subspace.
 func (k Keeper) AddUserToGroup(ctx sdk.Context, subspaceID uint64, groupID uint32, user string) {
+	// Create account if user does not exist.
```

Comment:
Should be this performed only if the subspace have a grant? If so we can create the account only if the subspace have an active grant  to reduce the data stored on disk.
If you think this is worth to do you should also update the `SaveGrant` inside `feegrant.go` to create the accounts when adding a feegrant to a group since can happen that some one added an address to a grop and then enabled a `GroupGrantee`.

Reply:
I think it is better to make users in the group always have their own account since the members should be considered to ready to performs transactions inside the subspace. As a result, the suggestion would not reduce the disk storage, and it would add more check when granting a group or add user into a group, like:
```go
func (k Keeper) AddUserToGroup(ctx sdk.Context, subspaceID uint64, groupID uint32, user string) {
	// Create account if member in the granted group does not exist.
	if k.HasGroupGrant(ctx, subspaceID, groupID) {
		k.createAccountIfNotExists(ctx, user)
	}
}

func  (k Keeper) SaveGrant(ctx sdk.Context, grant types.Grant) {
	...	
	if grantee, isGroupGrantee := grant.Grantee.GetCachedValue().(*types.GroupGrantee); isGroupGrantee {
		// Try to create accounts for members if the group is first granted
		if !k.HasGroupGrant(ctx, grant.SubspaceID, grantee.GroupID) {
			k.IterateUserGroupMembers(ctx, grant.SubspaceID, grantee.GroupID, func(member string) bool {
				k.createAccountIfNotExists(ctx, member)
				return false
			})
		}
	}
}
```

Please @RiccardoM I also would like to know what do you think about this.